op-challenger/game/keccak/types/types.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Leaf is the keccak state matrix added to the large preimage merkle tree.
+type Leaf struct {
+	// Input is the data absorbed for the block, exactly 136 bytes
+	Input [136]byte
```

Comment:
Here, you could just use `BlockSize` if it's defined in this source file I think (?).

Reply:
gah, there's always another raw 136 somewhere...  :D